src/js/lib/admin-page/style.scss

```diff
@@ -0,0 +1,165 @@
+$gray: #777;
+
+#wp-parsely-react-entrypoint {
+	.components-snackbar {
+		border-left: 5px solid green;
+		&.settings-error {
+			border-left: 5px solid red;
```

Comment:
Instead of just `green` and `red`, can you find the proper hex codes that match up to the wp-admin colour schemes? Let's try and make it fit in as much as possible.